docs/changelog.rst

```diff
@@ -12,6 +12,7 @@ Unreleased
 Added
 -----
 - **setup_ksatver_vegetation**: method to calculate KsatVer_vegetation to account for biologically-enhanced soil structure in KsatVer.
+- **setup_precip_from_point_timeseries**: method to interpolate rainfall station data as model forcing.
```

Comment:
Nice! Before it's finished, would be good to update the table with the available methods in the docs as well as the Model and workflows API docs.